docs: add docs about using secret (#909)
bytebase/bytebase.com: content/docs/administration/secret.md

---
title: Secret
---
<hint-block type="info">

This feature is only available in the Pro Plan and Enterprise Plan.

</hint-block>

Secrets are variables that you create in database. The secrets that you create are available to use in SQL change workflow.

## Naming your secrets

The following rules apply to secret names:

- Names can only contain uppercase alphanumeric (`[A-Z]`, `[0-9]`) or underscores (`_`). Spaces are not allowed.

- Names must not start with the `BYTEBASE_` prefix.

- Names must not start with a number.

- Names must be unique in a database.

## Creating secrets for a database

<hint-block type="info">

To create secretes for a database, you must be the Workspace Owner or Workspace DBA.

</hint-block>

### Step 1 - Navigate to Database

> In the Bytebase, navigate to detail page of the bytebase. Click `Settings` tab and you will see the Secret block.

Reviewer: ```suggestion
Navigate to the database detail page. Click `Settings` tab and you will see the Secret block.
```
